deployment/provider-vault-installer.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       tolerations:
       containers:
         - name: provider-vault-installer
-          image: hashicorp/secrets-store-csi-driver-provider-vault:0.0.1
+          image: hashicorp/secrets-store-csi-driver-provider-vault:0.0.2
```

Comment:
Should we update this file _after_ a new vault provider `0.0.3` has been released? I think it's currently at `0.0.2`

Reply:
It is currently at 0.0.2 and the deployment yaml wasn't up to date. So updated it to reflect the latest. Once `0.0.3` is cut, then this will need to be updated to show the latest image.

Reply:
okay cuz https://github.com/deislabs/secrets-store-csi-driver/blob/master/deploy/provider-vault.yaml#L21-L54 is  at `0.0.2` and it still has the provider-log stuff. I believe `0.0.2` still looks for that volume.

Reply:
I was hoping @malnick can get a new release once this PR is merged and we can just bump the version for the image. But I can add remove the `provider-log` when we have a new release.